BackEnd/tests/unit/test_s3.py

```diff
@@ -8,6 +8,7 @@
 BUCKET = 'houseit'
```

Comment:
maybe make all methods use file name since we don't have a different object name?